src/core/lib/gprpp/status_helper.cc

```diff
@@ -350,12 +350,25 @@ std::string StatusToString(const absl::Status& status) {
 
 namespace internal {
 
+// This is the key for the message but it's only used for serialization.
+constexpr char kMessageKey[] = "<m>";
```

Comment:
Suggest changing this to something like `"grpc.utf8_message"`, to make its purpose clearer.

Reply:
Done! But i removed utf8_ because it's not utf-8